lib/compiler/Cargo.toml

```diff
@@ -43,7 +44,7 @@ default = ["std", "enable-serde" ]
 # `CompilerConfig`, as well as the included wasmparser.
 # Disable this feature if you just want a headless engine.
 translator = ["wasmparser"]
-compiler = ["translator"]
+compiler = ["translator", "wasmer-object"]
```

Comment:
Why compiler requires `wasmer-object`? Should we have a new flag?

Reply:
Might be a mistake, will recheck